### PR TITLE
Feature/default sort criteria

### PR DESCRIPTION
**Pull request by melonistic**

### CODE REVIEW

**Comment by melonistic** [comment]
Test cases

```
TrackersController
  GET #index
    tracker order
      when the tracker order has been implemented
        order by protocol position
          expects tracker items to be sorted by protocol position in descending order
        order by event date
          expects tracker items to be sorted by event date in descending order
        order by protocol name
          expects tracker items to be sorted by protocol name
      when the tracker order has not been implemented
        expects the response to be a bad request

Finished in 30.58 seconds (files took 15.1 seconds to load)
4 examples, 0 failures

```

**Comment by melonistic** [comment]
How order-by is implemented for NULL is Postgres: https://www.postgresql.org/docs/8.3/queries-order.html
TL;DR for desc order, by default, nulls go first. for asc order, by default, nulls go last. In PG, nulls have a higher value than any value. In the case of event_dates, we have to explicitly set NULLS LAST since it is sorted in descending order.
